components/calculator/rows/row.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from "./row.module.css";
```

Comment:
This import should be removed if styles isn't used in your component or use the styles to style your component

Reply:
Will do, thanks